iot/http_example/system-test/cloudiot_http_example_test.js

```diff
@@ -0,0 +1,118 @@
+const cwd = path.join(__dirname, `..`);
+
+test.before(tools.checkCredentials);
+test.before(async () => {
+  const pubsub = PubSub();
+  return pubsub.createTopic(topicName)
+    .then((results) => {
+      const topic = results[0];
+      console.log(`Topic ${topic.name} created.`);
+      return topic;
+    });
+});
+
+test.after.always(async () => {
+  const pubsub = PubSub();
+  const topic = pubsub.topic(topicName);
+  return topic.delete()
+    .then(() => {
+      console.log(`Topic ${topic.name} deleted.`);
+    });
+});
+
+test(`should receive configuration message`, async (t) => {
+  const localDevice = `test-rsa-device`;
+  const localRegName = `${registryName}-rsa256`;
+  let output = await tools.runAsync(`${helper} setupIotTopic ${topicName}`, cwd);
+  output = await tools.runAsync(
+      `${helper} createRegistry ${localRegName} ${topicName}`, cwd);
+  output = await tools.runAsync(
+      `${helper} createRsa256Device ${localDevice} ${localRegName} resources/rsa_cert.pem`, cwd);
+  t.regex(output, new RegExp(`Created device`));
+
+  output = await tools.runAsync(
+      `${cmd} --message_type=events --num_messages=1 --private_key_file=resources/rsa_private.pem --algorithm=RS256`, cwd);
+
+  t.regex(output, new RegExp(`Getting config`));
+
+  // Check / cleanup
+  output = await tools.runAsync(
+      `${helper} getDeviceState ${localDevice} ${localRegName}`, cwd);
+  t.regex(output, new RegExp(`State`));
+  output = await tools.runAsync(
+      `${helper} deleteDevice ${localDevice} ${localRegName}`, cwd);
+  t.regex(output, new RegExp(`Successfully deleted device`));
+  output = await tools.runAsync(`${helper} deleteRegistry ${localRegName}`, cwd);
+});
+
+test(`should send event message`, async (t) => {
+  const localDevice = `test-rsa-device`;
+  const localRegName = `${registryName}-rsa256`;
+  let output = await tools.runAsync(`${helper} setupIotTopic ${topicName}`, cwd);
+  output = await tools.runAsync(
+      `${helper} createRegistry ${localRegName} ${topicName}`, cwd);
+  output = await tools.runAsync(
+      `${helper} createRsa256Device ${localDevice} ${localRegName} resources/rsa_cert.pem`, cwd);
+
+  output = await tools.runAsync(
+      `${cmd} --message_type=events --num_messages=1 --private_key_file=resources/rsa_private.pem --algorithm=RS256`, cwd);
+
+  t.regex(output, new RegExp(`Publishing message`));
+
+  // Check / cleanup
+  output = await tools.runAsync(
+      `${helper} getDeviceState ${localDevice} ${localRegName}`, cwd);
+  output = await tools.runAsync(
+      `${helper} deleteDevice ${localDevice} ${localRegName}`, cwd);
+  output = await tools.runAsync(`${helper} deleteRegistry ${localRegName}`, cwd);
+});
+
+test(`should send event message`, async (t) => {
+  const localDevice = `test-rsa-device`;
+  const localRegName = `${registryName}-rsa256`;
+  let output = await tools.runAsync(`${helper} setupIotTopic ${topicName}`, cwd);
+  output = await tools.runAsync(
```

Comment:
Is the `output = ` part required if you're not planning to use the output?

If not, please fix this throughout your PR.

Reply:
Done.